Test/datadirectorytest.py

```diff
@@ -152,5 +153,37 @@ def test_data_object(self):
         self.assertFalse(dd.is_file())
         self.assertTrue(dd.get_type() is DataObjectType.directory)
 
+class LocalDataDirectoryTest(unittest.TestCase):
```

Comment:
Does it make sense to move this to a separate file? The usual unittest pattern I see is 1 class per file; what are your thoughts on this?

Reply:
I would be totally open to either way.  Unless people feel strongly about it I will leave it as-is for now, but that's just because it's already written